core/services/database/TableManager.php

```diff
@@ -222,22 +211,49 @@ public function createTable($table_name, $create_sql, $engine = 'MyISAM')
                 )
             );
         }
+        $engine = apply_filters(
+            'FHEE__EventEspresso_core_services_database_TableManager__createTable__engine',
+            $engine,
+            $table_name,
+            $create_sql
+        );
+        $table_name = $this->getTableAnalysis()->ensureTableNameHasPrefix($table_name);
+        global $wpdb;
+        $collation = $wpdb->get_charset_collate();
+        $SQL = "
+CREATE TABLE {$table_name} (
+{$create_sql}
+) ENGINE={$engine}
+{$collation}
+";
```

Comment:
i did that so that the resulting SQL was formatted in a SQL-ish way, but can revert to having this code all in one line (although the resulting SQL will still be multiline)

Reply:
Oh, right, ok.

Again, maybe not worth spending much time on this peace, but normally I would deal with this like so:

```suggestion
        $SQL = "CREATE TABLE {$table_name} ("
                . "{$create_sql}"
                . ") ENGINE={$engine} "
                . "{$collation}";
```